src/style.css

```diff
@@ -1,3 +1,68 @@
+* {
+  margin: 0;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, serif;
+  font-weight: 500;
+  font-style: normal;
+
+  --main-color: #00acdc;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.nav__list {
+  display: flex;
+  padding: 0;
+}
+
+.nav__list__item {
+  display: inline-block;
+  text-transform: uppercase;
+  text-align: center;
+}
+
+.nav__link {
+  align-content: center;
```

Comment:
The `align-content` property is not applicable for inline-block elements like `.nav__link`. Consider removing it as it has no effect.